docs/ja/guides/pipeline-integrations/administration/setting-python-3-desktop.md

```diff
@@ -1,52 +1,52 @@
 ---
 layout: default
-title: デスクトップ用に Python 3 を設定する
+title: Desktop に Python 2 を設定する
```

Comment:
Is there any reason why lowering this to Python 2 instead?

Reply:
@carlos-villavicencio-adsk, the occurrences of trailing/leading whitespaces are because they have already existed in the source file. The localized files generated referring the source file. 
If these whitespaces should not exist in the files, vendors suggested removing them from the source file. Then they can be removed automatically from localized files.

Reply:
@carlos-villavicencio-adsk, about lowering the python version is also came from the source file.